climpyrical/tests/test_gridding.py

```diff
@@ -34,72 +36,72 @@ def test_check_ndims(data, n, passed, error):
             check_ndims(data, n)
 
 
-# create pseudo grids with expected dimension and ranges
-xi = np.linspace(-33.8800048828125, 33.8800048828125, 155)
-yi = np.linspace(-28.59999656677246, 28.15999984741211, 130)
-xext, yext = np.tile(xi, yi.size), np.repeat(yi, xi.shape)
-xext_bad, yext_bad = np.repeat(xi, yi.size), np.tile(yi, xi.shape)
+# load example ensemble dataset for testing
+ds = read_data(
+        resource_filename("climpyrical", "tests/data/snw_test_ensemble.nc"),
+        "Rain-RL50"
+    )
+
+# read grids with expected dimension and ranges
+xi, yi = ds.rlon.values, ds.rlat.values
+
+# extend coordinates in expected way
+xext_ex, yext_ex = np.tile(xi, yi.size), np.repeat(yi, xi.size)
+xext_bad, yext_bad = np.repeat(xi, yi.size), np.tile(yi, xi.size)
 
 
 @pytest.mark.parametrize(
-    "x,y,passed,error",
+    "x,y,ds,passed,error",
```

Comment:
Are you planning tests with varying `ds` parameters? If not, suggest you don't parametrize with it.

Also, your test parameter `ds` shadows the outer variable `ds`. Best to avoid that. This issue will not occur of course if `ds` parameter is removed.

Reply:
No parameterization. Thanks :+1: